### ISSUE TITLE
Does this work with twilio?

### ISSUE THREAD

**Issue by YpvzuBohdygxn9257** [stuntbanana/stuntbanana#3]

**Comment by maietta** [comment]
No. Twilio began restricting caller ID to only numbers you own or have verified. Any U.S. based VOIP or trunking service will have these restrictions, with many only allowing outgoing caller ID matching any number hosted on the service.

Based on the knowledge that you just created your account to post this issue with a random username, my guess is you are wanting to do something illegal with this, which is why I will not be recommending any companies that allow custom outgoing caller ID.

With the recent spike in swatting, I will not be disclosing any accumulated knowledge on the subject.

Please close this ticket.

**Comment by liho00** [comment]
@maietta I am researching this topic recently, I am wondering why not we host our own sip server for sip trunking, but instead we have to choose sip provider? would you like to explain to me? Thanks in advance!
